coordinator/scripts/calculate_results.py

```diff
@@ -326,6 +330,13 @@ def process_lats(lats):
                 elbow_latmean.append(prev_latmean)
                 elbow_lat99.append(prev_lat99)
                 elbow_lat99999.append(prev_lat99999)
+    elif 'TRIM_SAMPLES' in environ :
+        ## Lob off (configurable) more "warm up" samples
+        trim_samples = int(environ['TRIM_SAMPLES'])
+        for i in range(len(tps_lines)):
+            tps_lines[i]["tps"] = tps_lines[i]["tps"][trim_samples:]
+        for i in range(len(lat_lines)):
+            lat_lines[i]["lats"] = lat_lines[i]["lats"][trim_samples:]
```

Comment:
@maurermi my python is a bit rusty. I was expecting this code to trim the _first_ `trim_samples` samples from the set, but that doesn't appear to be what's happening when I go to test (the beginning of the test is preserved, and clearly the _last_ `trim_samples` samples are removed).

It's pretty clear the previous version of the code did exactly that, but I'm not sure why. Perhaps this should be merged (since it correctly applies the trimming to the stats and charts), and we should consider offering a manual trim-from-begin and trim-from-end inputs? (would mirror the auto-trim-zeros split well)